README.md

```diff
@@ -91,12 +90,12 @@ Let us know if we mistakenly label an idea as [unfunded](https://github.com/phot
 
 *Note that we've recently moved GitHub Sponsors from [sponsors/lastzero](https://github.com/sponsors/lastzero) to an 
 [organization account](https://github.com/sponsors/photoprism) so that donations can be shared more easily.
-Tiers are more affordable now while benefits stay mostly the same.*
+Tiers are more affordable now, while benefits stay mostly the same.*
 
 ## Trademarks ##
 
 PhotoPrism® is a registered trademark of Michael Mayer. You may use it as required to describe 
-our software, run your own server, for educational purposes, but not for offering commercial 
+our software, run your server for educational purposes, but not for offering commercial 
```

Comment:
These are two items: 1) run your server, 2) for educational purposes

Comma seems right?